pilot/pkg/bootstrap/server.go

```diff
@@ -605,7 +559,7 @@ func (s *Server) initSecureGrpcServer(port string, keepalive *istiokeepalive.Opt
 
 	tlsCreds := credentials.NewTLS(cfg)
 
-	// Default is 15012 - istio-agent relies on this as a default to distinguish what cert auth to expect
+	// Default is 15012 - istio-agent relies on this as a default to distinguish what cert auth to expect.
```

Comment:
Same - this was used for 1.4 - 1.5 upgrade, where we had 2 types of istiod certs. 
A TODO to cleanup agent startup will be good.